conftest.py

```diff
@@ -0,0 +1,4 @@
+import os
+import sys
+
+sys.path.append(os.path.join(os.curdir, "app"))
```

Comment:
This is considered a bad practice, and shouldn't be relevant if run from a different directory.

Reply:
ok, I didn't now this. I will fix this.
good to learn, thanks.